hack/commit-msg-verify.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
Only needed for prow, we can remove this

Reply:
We do have GH actions on this org right? We could call the script in the action :thinking:

Reply:
@gdbranco https://github.com/openshift-online/ocm-common/pull/3 -- added in this MR already :)